### PR TITLE
[Doc] reorganize show tablet and add return fields

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5
  - [x] 2.4

### CODE REVIEW

**Review by hellolilyliuyi — DISMISSED** [review]

File: docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md [hunk 1]
```diff
@@ -1,51 +1,220 @@
 # SHOW TABLET
 
-## Description
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md:3]: ## Description 这个header 现在不用了吗

File: docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md [hunk 2]
```diff
@@ -1,51 +1,220 @@
...
 ```sql
 SHOW TABLET
-[FROM [db_name.]table_name | tablet_id] [partition(partition_name_1, partition_name_1)]
-[where [version=1] [and backendid=10000] [and state="NORMAL|ROLLUP|CLONE|DECOMMISSION"]]
-[order by order_column]
-[limit [offset,]size]
+FROM [<db_name>.]<table_name>
+[PARTITION(<partition_name>, ...]
+[
+WHERE [<version = <version_number>]
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md:18]: 多了 `<`

File: docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md [hunk 3]
```diff
@@ -1,51 +1,220 @@
...
-[limit [offset,]size]
+FROM [<db_name>.]<table_name>
+[PARTITION(<partition_name>, ...]
+[
+WHERE [<version = <version_number>] 
+    [[AND] backendid = <backend_id>] 
+    [[AND] STATE = "NORMAL"|"ALTER"|"CLONE"|"DECOMMISSION"]
+]
+[ORDER BY <field_name> [ASC | DESC]]
+[LIMIT [<offset>,]<limit>];
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md:23]: 分号可删

File: docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md [hunk 4]
```diff
@@ -1,51 +1,220 @@
...
+| field_name     | No       | The field by which the results are sorted. All the fields returned by `SHOW TABLET FROM <table_name>` are sortable.<ul><li>If you want to display results in ascending order, use `ORDER BY field_name ASC`. </li><li>If you want to display results in descending order, use `ORDER BY field_name DESC`.</li></ul> |
+| offset         | No       | The number of tablets to skip from the results. For example, `OFFSET 5` means to skip the first five tablets. Default value: 0. |
+| limit          | No       | The number of tablets to return. For example, `LIMIT 10` means to return only 10 tablets. If this parameter is not specified, all the tablets that meet the filter conditions are returned. |
+
+### Query information of a single tablet
+
+After obtaining all tablet IDs using `SHOW TABLET FROM <table_name>`, you can query the information of a single tablet.
+
+```sql
+SHOW TABLET <tablet_id>;
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW TABLET.md:43]: 分号可删

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Pull request merged**